### PR TITLE
feat: remove label

### PR DESCRIPTION
**Pull request by Rudolphous**
Add functionality to remove a label from a given pull request.
Also converted fetch.js to unix format like the other files.

Changes have been tested and do work.

### CODE REVIEW

File: azure-api.js
Comment (pndewit) [inline azure-api.js:1]: Let's call the function `deleteLabel` as removing might imply the label is recoverable 😉

**Comment by pndewit** [comment]
Thank you for your contribution!

**Comment by Rudolphous** [comment]
You are welcome, looking forward for the release.

**Comment by pndewit** [comment]
2.1.0 has been released

**Pull request merged**